src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetBuilder.scala

```diff
@@ -0,0 +1,112 @@
+package io.iohk.ethereum.faucet.jsonrpc
+
+import java.security.SecureRandom
+
+import akka.actor.ActorSystem
+import io.iohk.ethereum.faucet.FaucetConfigBuilder
+import io.iohk.ethereum.jsonrpc.server.controllers.ApisBase
+import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcBaseController.JsonRpcConfig
+import io.iohk.ethereum.jsonrpc.server.http.JsonRpcHttpServer
+import io.iohk.ethereum.keystore.KeyStoreImpl
+import io.iohk.ethereum.mallet.service.RpcClient
+import io.iohk.ethereum.utils.{ConfigUtils, KeyStoreConfig, Logger}
+
+import scala.util.Try
+
+trait ActorSystemBuilder {
+  def systemName: String
+  implicit lazy val system: ActorSystem = ActorSystem(systemName)
```

Comment:
WDYT about passing dependencies as parameters to class rather than mixing traits?

I find mixin traits to be hard to test/mock/keep track to release properly/reason about.
Maybe it would be possible to pass ActorSystem as implicit param whenever it is needed?